apps/api/src/domain/models/user-management/group/entities/coscrad-user-group.entity.ts

```diff
@@ -53,4 +59,30 @@ export class CoscradUserGroup extends Aggregate {
         // There are no complex invariant rules for a `CoscradUserGroup`
         return Valid;
     }
+
+    validateExternalState({
```

Comment:
Add a story to add a unit test for this.

It's not super urgent as the command test serves as an integration test for this logic.

Reply:
added [this story](https://www.pivotaltracker.com/story/show/182727483) and linked.